app/features/welcome/components/Welcome.js

```diff
@@ -104,12 +128,31 @@ class Welcome extends Component<Props, State> {
                         <Body>
                             <RecentList />
                         </Body>
+                        { this._renderActiveOnboardingComponent() }
                     </Wrapper>
                 </AtlasKitThemeProvider>
             </Page>
         );
     }
 
+    /**
+     * Render active onboarding component.
+     *
+     * @returns {void}
+     */
+    _renderActiveOnboardingComponent() {
```

Comment:
I think this could be put into a `OnboardingContainer` or just `Onboarding` component, which gets a prop with the section name. That way it can be reused in both locations.

Reply:
Nice catch.
I tried that when we didn't have sections for onboarding but I guess I just forgot. :smile: